src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -135,6 +135,9 @@ private static bool ApplyChangesFromTC32(JObject sarifLog)
                 {
                     // https://github.com/oasis-tcs/sarif-spec/issues/325
                     RemoveToolLanguage(run);
+
+                    // Modify reportingDescriptor.name type to string
```

Comment:
>// Modify reportingDescriptor.name type to string [](start = 20, length = 49)

@mikefan@microsoft.com i couldnt spot the right issue number in sarif-issues list to update here and the release notes md. Can you point to the right item?

Reply:
These changes are part of the localization removal issue, see below.

[https://github.com/oasis-tcs/sarif-spec/issues/325](https://github.com/oasis-tcs/sarif-spec/issues/325)

---
In reply to: [260425123](https://github.com/Microsoft/sarif-sdk/pull/1309#discussion_r260425123) [](ancestors = 260425123)

Reply:
thanks! will update

---
In reply to: [260549244](https://github.com/Microsoft/sarif-sdk/pull/1309#discussion_r260549244) [](ancestors = 260549244,260425123)

Reply:
we have an entry for this in release notes, thanks much!

---
In reply to: [260551276](https://github.com/Microsoft/sarif-sdk/pull/1309#discussion_r260551276) [](ancestors = 260551276,260549244,260425123)